modules/govuk_jenkins/templates/jobs/copy_data_from_production_to_aws.yaml.erb

```diff
@@ -0,0 +1,111 @@
+---
+- scm:
+    name: env-sync-and-backup_Copy_Data_to_Production
+    scm:
+        - git:
+            url: git@github.com:alphagov/env-sync-and-backup.git
+            branches:
+              - master
+
+- job:
+    name: Copy_Data_from_Production_to_AWS
+    display-name: Copy_Data_from_Production_to_AWS
+    project-type: freestyle
+    description: |
+        This job copies databases from production to AWS. It runs nightly and
+        can be run in an ad-hoc fashion. It doesn't copy the signon or
+        performance-platform databases.
+    properties:
+        - github:
+            url: https://github.com/alphagov/env-sync-and-backup/
+        - inject:
+            properties-content: |
+              PARALLEL_JOBS=3
+        - slack:
+            notify-start: false
+            notify-success: true
+            notify-aborted: true
+            notify-notbuilt: true
+            notify-unstable: false
+            notify-failure: true
+            notify-backtonormal: false
+            notify-repeatedfailure: false
+            include-test-summary: false
+            room: <%= @slack_room %>
+    scm:
+      - env-sync-and-backup_Copy_Data_from_Production_to_AWS
+    logrotate:
+        numToKeep: 10
+    triggers:
+        - timed: 'H 1 * * *'
```

Comment:
Will this happen at the same time as the current data sync?

Reply:
Good point. The current Integration data sync is set to happen at `H 4 * * 1-5`, and the current Staging one set to happen at `H 1 * * *`, so yes. The Integration AWS sync was set to happen at `H 11 * * 1-5`. I'll make this data sync consistent with the latter.